Source/Diagnostics/ReducedDiags/FieldMaximum.cpp

```diff
@@ -62,6 +62,14 @@ FieldMaximum::FieldMaximum (const std::string& rd_name)
     // resize data array
     m_data.resize(noutputs*nLevel, 0.0_rt);
 
```

Comment:
Hm, in this file, for `WARPX_DIM_RZ`, we do not do anything specific (and only raise an error message). Should we do the same here for `WARPX_DIM_RCYLINDER` and `WARPX_DIM_RSPHERE`?

Reply:
Similar to what is done in `Source/Diagnostics/ReducedDiags/FieldMomentum.cpp`